clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -304,7 +310,9 @@ public void testDelete_FileExists() throws ApiException, IOException {
                     .thenReturn(new ObjectStatsList().results(Collections.emptyList()).pagination(new Pagination().hasMore(false)));
         }
         StagingLocation stagingLocation = new StagingLocation().token("foo").physicalAddress(s3Url("/repo-base/dir-marker"));
-        when(stagingApi.getPhysicalAddress("repo", "main", "no/place/", false))
+        when(stagingApi.getPhysicalAddress("repo", "main", "no/place/"))
+                .thenReturn(getPhysicalAddressRequest);
+        when(getPhysicalAddressRequest.execute())
```

Comment:
AFAICT these calls show that `getPhysicalAddressRequest` should be a local variable here rather than a member.  Consider a test that causes `getPhysicalAddress` to run twice.

Reply:
Yeah, you're right. However, I don't want to invest more time into this before we decide if we want to move forward with this approach. If we do, I'll fix the test as you suggested.